FWCore/MessageService/src/ELoutput.h

```diff
@@ -98,6 +98,7 @@ namespace edm {
       bool wantTimestamp, wantModule, wantSubroutine, wantText, wantSomeContext, wantSerial, wantFullContext,
           wantTimeSeparate, wantEpilogueSeparate, preambleMode;
 
+    public:
       // --- Verboten method:
       //
       ELoutput& operator=(const ELoutput& orig) = delete;
```

Comment:
I'd prefer to move this to the top in the existing `public:` section.

Reply:
sure I will take care of these. I just wanted to make minimal changes :-) I will update the PR to move these deleted constructor to top